.travis.yml

```diff
@@ -1,12 +1,8 @@
 language: scala
 scala:
-  - 2.10.4
-  - 2.11.1
   - 2.11.7
-script:
-  - sbt ++$TRAVIS_SCALA_VERSION scoverage:test coveralls
 jdk:
```

Comment:
Travis is calling `sbt test` by default, but there is need to call `scalastyle` also. Probably you should add:

```
script:
  - sbt compile test scalstyle
```